<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QWidget" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>music</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>20</width>
     <height>601</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>41</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="modeButton_2">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>11</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{border-image: url(:/22.png);}



</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="preButton">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>11</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{border-image: url(:/5.png);}
QPushButton:hover{border-image: url(:/14.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton_4">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>11</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
border-image: url(:/1.png);
background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
image: url(:/13.png);
background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton_3">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>11</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/7.png);}
QPushButton:hover{border-image: url(:/15.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="voice">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/52.png);}
QPushButton:hover{border-image: url(:/21.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>modeButton_2</zorder>
   <zorder>preButton</zorder>
   <zorder>nextButton_3</zorder>
   <zorder>voice</zorder>
   <zorder>playButton_4</zorder>
  </widget>
  <widget class="QPushButton" name="recentPlay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>261</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 5);
border-radius:15px;
font: 290 16pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
   </property>
   <property name="text">
    <string>最近播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>261</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 5);
border-radius:15px;
font: 290 16pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
   </property>
   <property name="text">
    <string>本地与下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loveMusic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>261</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 5);
border-radius:15px;
font: 290 16pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
   </property>
   <property name="text">
    <string>我喜欢的音乐</string>
   </property>
  </widget>
  <widget class="QLabel" name="lovePic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>267</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/17.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="downloadPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/18.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rencentPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/16.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundPic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/49.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="playslider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>490</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal{
        border-radius:0px;
		width: 10px; 
	    background-color: rgba(255, 255, 255, 0);
        margin: -5px -5px -5px -5px;
} 

QSlider::handle:horizontal:hover{ 
		width:  15px; 
 		border-image:url(:/53.png);
        margin: -5px -5px -5px -5px;
} 
QSlider::groove:horizontal{ 
		height: 6px; 
		left: 0px; 
		right: 0px; 
		border:0px;
	    border-radius:6px;
		background:rgba(0,0,0,50);
 } 
QSlider::sub-page:horizontal{
	background-color: rgba(18, 146, 214, 180);
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>76</y>
     <width>731</width>
     <height>399</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="downloadPage">
    <widget class="QPushButton" name="localSings">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>220</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 50);
border-radius:0px;
font: 290 12pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
     </property>
     <property name="text">
      <string>本地歌曲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadedSings">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>220</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 50);
border-radius:0px;
font: 290 12pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
     </property>
     <property name="text">
      <string>下载歌曲</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>74</y>
       <width>731</width>
       <height>320</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>259</x>
         <y>100</y>
         <width>161</width>
         <height>111</height>
        </rect>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/23.png);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>78</x>
          <y>40</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;添加&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>3</y>
          <width>151</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 100);
border-radius:30px;
font: 290 16pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_2">
      <widget class="QListWidget" name="musicList">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>26</y>
         <width>661</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{
background-color:rgba(177, 177, 177, 100);
border-radius:0px;
font: 700 16pt &quot;微软雅黑&quot;;
}
QListWidget::Item{
	height:40px;
}</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>661</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>3295</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(177, 177, 177, 100);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>  歌曲</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>324</x>
          <y>0</y>
          <width>331</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(177, 177, 177, 0);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>歌手</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>661</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>3295</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(177, 177, 177, 100);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>  歌曲</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>324</x>
          <y>0</y>
          <width>331</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(177, 177, 177, 0);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>歌手</string>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="musicList2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>26</y>
         <width>661</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{
background-color:rgba(177, 177, 177, 100);
border-radius:0px;
font: 700 16pt &quot;微软雅黑&quot;;
}
QListWidget::Item{
	height:40px;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="addWidget" native="true">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>10</y>
       <width>225</width>
       <height>65</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/23.png);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addAgainButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>64</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 50);
border-radius:0px;
font: 290 12pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="musicHomePage"/>
   <widget class="QWidget" name="rencentpage"/>
   <widget class="QWidget" name="lovePage"/>
  </widget>
  <widget class="mySlider" name="voiceSlider">
   <property name="geometry">
    <rect>
     <x>879</x>
     <y>533</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{ 
		height: 6px; 
		left: 0px; 
		right: 0px; 
		border:0px;
	    border-radius:6px;
		background:rgba(0,0,0,50);
 } 
    
QSlider::handle:horizontal{ 
        border-radius:0px;
		width:  15px; 
 		border-image:url(:/53.png);
        margin: -5px -5px -5px -5px;
} 
QSlider::sub-page:horizontal{
	background-color: rgba(18, 146, 214, 180);
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>470</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>470</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coverArtImage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>69</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border-image:  url(:/24.png);}
QPushButton:hover{
border-image: url(:/32.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="maxButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{border-image:  url(:/25.png);}
QPushButton:hover{
border-image: url(:/31.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border-image:  url(:/26.png);}
QPushButton:hover{
border-image: url(:/30.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>261</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgba(255, 255, 255, 5);
border-radius:15px;
font: 290 16pt &quot;微软雅黑 Light&quot;;
}

QPushButton:hover{background-color: rgba(177, 177, 177, 100);}

</string>
   </property>
   <property name="text">
    <string>音乐馆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>219</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;
background-color: rgba(0, 85, 255, 50);</string>
   </property>
   <property name="text">
    <string>我的音乐</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;
background-color: rgba(0, 85, 255, 50);</string>
   </property>
   <property name="text">
    <string>在线音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backgroundChangeButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border-image:  url(:/45.png);}
QPushButton:hover{
border-image: url(:/46.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>202</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit{
background-color: rgba(255, 255, 255, 100);
border-radius:5px;
font: 290 12pt &quot;微软雅黑 Light&quot;;
}

QLineEdit:hover{background-color: rgba(177, 177, 177, 100);}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>13</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border-image: url(:/54.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundPic</zorder>
  <zorder>line_2</zorder>
  <zorder>widget1</zorder>
  <zorder>recentPlay</zorder>
  <zorder>download</zorder>
  <zorder>loveMusic</zorder>
  <zorder>lovePic</zorder>
  <zorder>downloadPic</zorder>
  <zorder>rencentPic</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>positionLabel</zorder>
  <zorder>durationLabel</zorder>
  <zorder>coverArtImage</zorder>
  <zorder>closeButton</zorder>
  <zorder>maxButton</zorder>
  <zorder>minButton</zorder>
  <zorder>line</zorder>
  <zorder>musicHome</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>backgroundChangeButton</zorder>
  <zorder>voiceSlider</zorder>
  <zorder>playSlider</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
  <customwidget>
   <class>playslider</class>
   <extends>QSlider</extends>
   <header>playslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
